concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -584,3 +591,86 @@ def test_provider_send_force_payment_with_same_subtasks_id_concent_should_refuse
             }
         )
         self._assert_stored_message_counter_not_increased()
+
+    def test_sum_of_payments_return_diffrent_value_then_return_from_mocked_backend(self):
```

Comment:
We need also test when `amount_pending` is 0.

Reply:
Done